### ISSUE TITLE
Some characters are transparent when using Noto Sans

### ISSUE THREAD

**Issue by vinibanaco** [diegomura/react-pdf#1303]
**Describe the bug**
Some characters are transparent when using Noto Sans:

![image](https://user-images.githubusercontent.com/44006807/117325555-3345a980-ae67-11eb-9c81-bb11368db21e.png)

The texts used in the above example were:
```
AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz
abcdefghijklmnopqrstuvwxyz
ABCDEFGHIJKLMNOPQRSTUVWXYZ
```

_I also tested with Roboto and it worked perfectly._

Some peculiarities that I noticed:

- You can select and copy the transparent characters as normal
- Numbers are always transparent
- If the text keeps the same, you can generate the PDF how many times you want, the transparent characters don't change
- Changing the text can change the transparent characters, specially the first line. On the screenshot below I sent the first line to the last position and this happened:

![image](https://user-images.githubusercontent.com/44006807/117330102-bb2db280-ae6b-11eb-8718-c1c3cd36a0a3.png)

**To Reproduce**
1. Download the font family from [Google Fonts](https://fonts.google.com/specimen/Noto+Sans)
2. Extract it in your project
3. Create a PDF that uses this font:

```jsx
import React from 'react';
import { Page, Font, Text, Document } from '@react-pdf/renderer';
import notoSansRegular from '../../../../../assets/fonts/NotoSans/NotoSans-Regular.ttf';

Font.register({
  family: 'NotoSans',
  src: notoSansRegular,
});

export default function Pdf() {
  return (
    <Document>
      <Page size="A4">
        <Text style={{ fontFamily: 'NotoSans' }}>AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz</Text>
        <Text style={{ fontFamily: 'NotoSans' }}>abcdefghijklmnopqrstuvwxyz</Text>
        <Text style={{ fontFamily: 'NotoSans' }}>ABCDEFGHIJKLMNOPQRSTUVWXYZ</Text>
      </Page>
    </Document>
  );
}
```

*Obs.: the result is the same when using a CDN (like [this one](https://cdn.jsdelivr.net/npm/notosans-fontface@1.2.2/fonts/NotoSans-Regular.ttf)), but the transparent characters change.

**Desktop:**
 - OS: Linux Mint 20
 - Browser: Chrome 90.0.4430.93
 - React-pdf version: 1.6.15

**Comment by diegomura** [comment]
Fixed in 2.0. see https://github.com/diegomura/react-pdf/issues/1194

**Comment by vinibanaco** [comment]
Oh nice! But sadly I won't be able to update because of #1210
